elastalert/zabbix.py

```diff
@@ -0,0 +1,102 @@
+from alerts import Alerter  # , BasicMatchString
+import logging
+# from pprint import pprint
+from pyzabbix.api import ZabbixAPI
+from pyzabbix import ZabbixSender, ZabbixMetric
+# from pyzabbix.api import ZabbixAPIException
+# from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from datetime import datetime
+
+
+class ZabbixClient(ZabbixAPI):
+
+    def __init__(self, url='http://localhost', use_authenticate=False, user='Admin', password='zabbix', sender_host='localhost',
+                 sender_port=10051):
+        self.url = url
+        self.use_authenticate = use_authenticate
+        self.sender_host = sender_host
+        self.sender_port = sender_port
+        self.metrics_chunk_size = 200
+        self.aggregated_metrics = []
+        self.logger = logging.getLogger(self.__class__.__name__)
+        super(ZabbixClient, self).__init__(url=self.url, use_authenticate=self.use_authenticate, user=user, password=password)
+
+    def send_metric(self, hostname, key, data):
+        zm = ZabbixMetric(hostname, key, data)
+        if self.send_aggregated_metrics:
+
+            self.aggregated_metrics.append(zm)
+            if len(self.aggregated_metrics) > self.metrics_chunk_size:
```

Comment:
What happens if this *isn't* true?

Reply:
The metric is appended (line 28, line 25 after remove imports commented), it isn't lost. When the number of metrics appended is greather than the metrics_chunk_size, the metrics are sent by ZabbixSender (line 32, line 29 after remove imports commented)

Reply:
@Qmando what do you think about the PR? maybe I can setup a var to manage the metrics_chunk_size in order to control the shipments to zabbix with it.